Hunted Mobile/Hunted Mobile/Hunted Mobile/View/GadgetsPage.xaml

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:viewModels="clr-namespace:Hunted_Mobile.ViewModel"
+             x:Class="Hunted_Mobile.View.GadgetsPage">
+    <AbsoluteLayout>
+        <!-- Main layout -->
+        <Grid x:Name="mainLayout">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="75" />
+            </Grid.RowDefinitions>
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition />
+                <ColumnDefinition />
+            </Grid.ColumnDefinitions>
+
+            <!-- Hunted title -->
+            <Label Text="Gadgets" 
+                   Padding="5" 
+                   HorizontalTextAlignment="Center" 
+                   TextColor="{AppThemeBinding Light=Black, Dark=White}" 
+                   FontAttributes="Bold"
+                   Grid.ColumnSpan="2"
+                   FontSize="36"/>
+
+            <ListView Grid.Row="1" ItemsSource="{Binding Gadgets}">
+                <ListView.ItemTemplate>
+                    <DataTemplate>
+                        <ViewCell>
+                            <StackLayout Padding="10">
+                                <Label Text="{Binding Name}" HorizontalTextAlignment="Center"/>
```

Comment:
Wil je ook niet icoontjes van gadgets weergeven? Of zal ik dat tegelijk doen met de beschrijving?